test/ctr.bats

```diff
@@ -12,6 +12,10 @@ function teardown() {
 	cleanup_test
 }
 
+function is_v2() {
```

Comment:
nit: Maybe rename this to `is_cgroupsv2`?

Reply:
sure

Reply:
I'd prefer `is_cgroupv2` though

Reply:
fixed; hope I didn't break anything this time :crossed_fingers: